mail/templatetags/timeago.py

```diff
@@ -17,3 +21,15 @@ def parse_iso(value):
         datetime: the parsed datetime
     """
     return parse(value)
+
+
+@register.filter
+def absolutize_profile_image_url(url):
+    """
+    Takes a URL, and if it is the default URL makes
+    it an absolute URL
+    """
+
+    if url == default_profile_image:
+        return urljoin(SITE_BASE_URL, url)
```

Comment:
Suggestion: could we instead move this `urljoin` into the `get_profile_image` method of the serializer?

Reply:
sure